CaliptraTrademarkPolicy.md

```diff
@@ -0,0 +1,112 @@
+# Caliptra Trademark Usage Policy
+
+* Authors: Hilary Richardson, Dan Scales
+* Reviewed by: Caliptra TAC founding members
+* Last major revision: March 30th 2023
+
+The Caliptra project trademarks and logos (“Caliptra Marks”) are licensed
+separately from any copyright and patent license grants contained in the
+Caliptra project GitHub repositories’ applicable [open source
+licenses](LICENSE). Except as provided in this policy and under fair use
+exceptions, you may not use the Caliptra Marks without permission.
+
+## Purpose of the Trademark Usage Policy
+
+This legal policy supplements the [Linux Foundation Trademark Usage
+policy](https://www.linuxfoundation.org/legal/trademark-usage) and
+contains important information. As the steward for the Caliptra silicon root of
+trust project, the Linux Foundation holds the Caliptra Marks. This Caliptra
+Trademark Policy exists to protect the work of the Caliptra project and the
+users of this technology. It ensures that Caliptra users and developers are not
+harmed through misrepresentation of the Caliptra project.
+
+In order to succeed in the project goals to increase security for users of
+Caliptra implementations and maintain hard-earned trust, the Caliptra name must
+be used carefully. Users should know that a Caliptra or Caliptra project-based
+product using the Caliptra brand equates to quality and security.
+
+## Acceptable Use
+
+You may use the Caliptra Marks without prior written permission (subject to the
+rest of this policy):
+
+1. To refer to unmodified source code, register transfer language, or other files
```

Comment:
If referring to the hardware description, RTL is "register-transfer level".

Reply:
Thanks, fixed